ddtrace/contrib/starlette/__init__.py

```diff
@@ -0,0 +1,69 @@
+"""
+The Starlette integration will trace requests to and from Starlette.
+
+
+Enable Starlette tracing by using the autopatching functionality::
+
+    from starlette.applications import Starlette
+    from ddtrace import tracer
+    from ddtrace.contrib.starlette import patch
+
+    patch()
+
+    app = Starlette()
+
+
+If using Python 3.6, the legacy ``AsyncioContextProvider`` will have to be
+enabled before using the middleware::
+
+    from ddtrace.contrib.asyncio.provider import AsyncioContextProvider
+    from ddtrace import tracer  # Or whichever tracer instance you plan to use
+    tracer.configure(context_provider=AsyncioContextProvider())
+
+
+Configuration
+~~~~~~~~~~~~~
+
+.. py:data:: ddtrace.config.starlette['distributed_tracing_enabled']
```

Comment:
```suggestion
.. py:data:: ddtrace.config.starlette['distributed_tracing']
```

Might also be worth including the environment variable here too.